composable_kernel/include/tensor_operation/gridwise_dynamic_gemm_xdlops_v2r3.hpp

```diff
@@ -73,8 +74,9 @@ __global__ void
         *reinterpret_cast<const AK0MK1GridDesc*>((const void*)p_a_k0_m_k1_grid_desc);
     const auto b_k0_n_k1_grid_desc =
         *reinterpret_cast<const BK0NK1GridDesc*>((const void*)p_b_k0_n_k1_grid_desc);
-    const auto c_m0_m1_m2_n_grid_desc =
-        *reinterpret_cast<const CM0M1M2NGridDesc*>((const void*)p_c_m0_m1_m2_n_grid_desc);
+    const auto c_m0_n0_m1_n1_m2_m3_m4_n2_grid_desc =
+        *reinterpret_cast<const CM0N0M1N1M2M3M4N2GridDesc*>(
```

Comment:
Pointer cast for vector type has been update in https://github.com/ROCmSoftwarePlatform/composable_kernel/pull/8

reinterpret_cast for vector type is undefined behavior